install/error-handling.sh

```diff
@@ -1,10 +1,34 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://afa81b4b7c634e318b0ec138abb2645f@self-hosted.getsentry.net/2'
+export SENTRY_ORG=sentry
+export SENTRY_PROJECT=dogfooding
```

Comment:
Really? :D Why not `self-hosted`?

Reply:
It's already at `self-hosted.sentry.net`! But I can always change this if you think I should.

Reply:
Ah, didn't notice that :D I'd still prefer something like `installer` or something but of course, up to you :)

Reply:
I like `installer`, fits if we ever get around to adding run-time monitoring in a separate project.

Reply:
Done!